csrc/device_lower/validation.cpp

```diff
@@ -201,6 +201,27 @@ void validateCpAsyncBulk(const std::vector<TensorView*>& tvs) {
           "Invalid iter type for cp.async.bulk: ",
           id->getIterType());
     }
+
+    // For 1D TMA, ID parallelized by Bulk can't comes from non-divisible split
```

Comment:
See my comment for predicate.cpp. This may not be necessary if `GpuLower::validate` is used.

Reply:
Good point, I am not aware of this runtime validate feature.